core/src/main/java/org/dcsa/conformance/core/check/JsonSchemaValidator.java

```diff
@@ -28,7 +31,7 @@ private JsonSchemaValidator(String filePath, String schemaName) {
     InputStream schemaFileInputStream = JsonSchemaValidator.class.getResourceAsStream(filePath);
     JsonSchemaFactory jsonSchemaFactory =
         JsonSchemaFactory.builder(JsonSchemaFactory.getInstance(SpecVersion.VersionFlag.V7))
-            .objectMapper(new ObjectMapper(new JsonFactory()))
+            .objectMapper(JSON_FACTORY_OBJECT_MAPPER)
```

Comment:
¯\_(ツ)_/¯ If this is important to you, please do the research and change it in a later PR.

I left it as-is because it was easier. Note it is specifically the `new ObjectMapper()` (argument less) that I was going for. I happened to notice this one and take it along, but I can revert it back to status quo if you prefer while you research.